site/layouts/_default/single.html

```diff
@@ -1,7 +1,11 @@
 {{ define "main" }}
 
     {{ partial "jumbotron" (dict  "title" .Title "subtitle" .Params.subtitle) }}
-    <div class="cms ph3 mw7 center">
+<<<<<<< Updated upstream
```

Comment:
this is one of the trickier things about git, there was a merge conflict and these fishbone/conflict markers need to get removed. We can discuss on Zoom and I can help with resolving it.